src/main/java/seedu/address/logic/commands/EditCommand.java

```diff
@@ -63,6 +63,14 @@ public EditCommand(Index index, EditEntryDescriptor editEntryDescriptor) {
         this.entryType = editEntryDescriptor.getType().get();
     }
 
+    private Entry getEntryToEdit(Index targetIndex, List<Entry> lastShownList)
```

Comment:
Added this method so it's easier to draw in the sequence diagram, but can we add private methods to the sequence diagram?

Reply:
Im unable to find private methods in the sequence diagrams in our textbook, I searched online as well and they mentioned it is okay to put if we want the developers to understand our implementation, but it is also okay not to put.

What do the rest think?

@AY2223S1-CS2103T-W17-2/developers